docs/reference/cli.md

```diff
@@ -726,14 +730,19 @@ View a lineage record.
 $ nextflow lineage view <lid>
 ```
 
+:::{tip}
+You can use the [jq](https://jqlang.org/) command-line tool to apply further queries and transformations on the resulting lineage record.
+:::
+
 The output of a workflow run can be shown by appending `#output` to the workflow run LID:
 
 ```console
 $ nextflow lineage view lid://c914d714877cc5c882c55a5428b510b1#output
 ```
 
-:::{tip}
-You can use the [jq](https://jqlang.org/) command-line tool to apply further queries and transformations on the resulting lineage record.
+:::{warning}
+Only outputs described using the {ref}`workflow-outputs` syntax are captured in lineage records.
+If your pipeline only uses {ref}`publishDir` then an empty list will be returned.
```

Comment:
```suggestion
If your pipeline only uses {ref}`publishDir` then an empty list will be returned.
However, files published via `publishDir` can be accessed using the Lineage ID in the format `lid://<workflow_lid>/<relative_path>`, where `<relative_path>` refers to the published file's path relative to the workflow's output directory. If the `publishDir` path is not a subpath of the output directory, the lineage metadata entry for these outputs will not be created, and a warning message will be printed. 
```

Reply:
Great - I wonder if it would make sense to mention that the base directory can be set with a config one-liner using `params.outdir`? This is also maybe getting a bit too long for an admonition.

I'll leave this for @christopher-hakkaart to make a decision on / clean up, if that's ok.

Reply:
I've edited this comment for language, please check the meaning is retained.